ui/component/optimizedImage/index.js

```diff
@@ -0,0 +1,6 @@
+import { connect } from 'react-redux';
+import OptimizedImage from './view';
+
+const perform = (dispatch) => ({});
+
+export default connect(null, perform)(OptimizedImage);
```

Comment:
connect has nothing to do here.
index.js  can just import and export the default from ./view
Though, I sometimes do this if I expect to add some selectors or actions later.